e2e/test/scenarios/admin/performance/preemptiveCaching.cy.spec.ts

```diff
@@ -0,0 +1,136 @@
+const { H } = cy;
+
+import { WRITABLE_DB_ID } from "e2e/support/cypress_data";
+import { createQuestion, getTableId, visitQuestion } from "e2e/support/helpers";
+
+import { TEST_TABLE, sampleNativeQuestion } from "./helpers/constants";
+import {
+  advanceServerClockBy,
+  interceptPerformanceRoutes,
+  resetServerTime,
+  setupQuestionTest,
+} from "./helpers/e2e-performance-helpers";
+import {
+  durationRadioButton,
+  enablePreemptiveCaching,
+  openSidebarCacheStrategyForm,
+  saveCacheStrategyForm,
+} from "./helpers/e2e-strategy-form-helpers";
+
+const CACHE_DURATION_MS = 24 * 60 * 60 * 1000;
+
+const WRITABLE_TEST_TABLE = "scoreboard_actions";
+
+describe(
+  "Preemptive caching for MBQL and native queries",
+  { tags: "@external" },
+  () => {
+    describe("ee", () => {
+      beforeEach(() => {
+        resetServerTime();
+        interceptPerformanceRoutes();
+        H.restore("postgres-writable");
+        cy.signInAsAdmin();
+        H.setTokenFeatures("all");
+        H.resetTestTable({ type: "postgres", table: WRITABLE_TEST_TABLE });
+        H.resyncDatabase({
+          dbId: WRITABLE_DB_ID,
+          tableName: WRITABLE_TEST_TABLE,
+        });
+      });
+
+      it("Returns matching results when an MBQL query is cached preemptively", () => {
+        // Using WRITABLE_TEST_TABLE because creating a question using TEST_TABLE doesn't work...
```

Comment:
I spent a while trying to get `TEST_TABLE` to work here since that's what's used in the other caching tests, but for some reason it fails the `createQuestion` call. It also doesn't work in `resyncDatabase` for some reason. Still not sure why, but switching to using `WRITABLE_TEST_TABLE` does work.